libs/langchain/langchain/vectorstores/pgvector.py

```diff
@@ -168,7 +169,29 @@ def delete_collection(self) -> None:
                 return
             session.delete(collection)
             session.commit()
-
+            
+    def delete(
+        self,
+        ids: Optional[List[str]] = None,
```

Comment:
`ids` should be used to delete `custom_id`  to be consistent with other vectorstores. the `id` is the user assigned id for the document

If we want the ability to delete by `uuid`, we can rename `custom_ids` to `uuids` and have it delete by uuid

Reply:
Thanks! I'm merging and going to make a PR to make the deletion more efficient. Deletion can be made much faster, but using a single SQL request.